PGIP/GraphQL/Resolver/ToResult.hs

```diff
@@ -0,0 +1,405 @@
+module PGIP.GraphQL.Resolver.ToResult where
+
+import PGIP.GraphQL.Result.Axiom as GraphQLResultAxiom
```

Comment:
maybe you can make the code shorter by omitting `GraphQLResult` from the qualified module names.

Reply:
This may clash with `Axiom` from `Persistence.Schema`. I chose to qualify both modules to avoid misunderstandings.